pyproject.toml

```diff
@@ -36,12 +36,12 @@ dependencies = [
 dev = [
   "ariadne",
   "pytest",
-  "pylint",
+  "pylint==3.2.7",
```

Comment:
Pinned due to [a bunch of fixes and changes in 3.3.0](https://github.com/pylint-dev/pylint/releases/tag/v3.3.0)